### PR TITLE
[bitnami/redis] add sampleLimit and targetLimit for redis chart

### PR DESCRIPTION
**Pull request by mblaschke**
### Description of the change
add `sampleLimit` and `targetLimit` for `ServiceMonitor`:
https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#servicemonitor

### Benefits

able to control the maximum number for sampleLimits and in some environments `sampleLimit` is a required field

### Possible drawbacks

none
### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/redis/Chart.yaml [hunk 1]
```diff
@@ -25,4 +25,4 @@ maintainers:
 name: redis
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/redis
-version: 17.12.0
+version: 17.12.1
```
Comment (aoterolorenzo) [inline bitnami/redis/Chart.yaml:28]: As this is a new feature we would bump a minor version following [semver](https://semver.org/): `17.13.0`. Could you update it?

**Review by aoterolorenzo — CHANGES_REQUESTED** [review]
@mblaschke thanks for the contribution!

Overall LGTM, but I left you a minor comment regarding the versioning.

File: bitnami/redis/Chart.yaml (hunk 1 above)
Comment (mblaschke) [reply bitnami/redis/Chart.yaml]: @aoterolorenzo done

**Review by aoterolorenzo — APPROVED** [review]
@mblaschke LGTM! Thanks for the contribution

**Pull request merged**